apis/r/tests/testthat/test-06-SOMADataFrame.R

```diff
@@ -501,13 +500,6 @@ test_that("creation with ordered factors", {
   expect_identical(lvls$ord, levels(df$ord))
   expect_identical(sdf$levels("ord"), levels(df$ord))
 
-  expect_s3_class(
-    ord <- sdf$.__enclos_env__$private$.tiledb_array[]$ord,
-    c("ordered", "factor"),
-    exact = TRUE
-  )
-  expect_length(ord, n)
-  expect_identical(levels(ord), levels(df$ord))
```

Comment:
This should be updated to do a read of the `"ord"` column and check that the data is being round-tripped as `"ordered"`